HW_dev_pro/education_terraform/HW_code_with_modules/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "image_id" {
```

Comment:
we should always grub latest ami from aws, not a permanent for better updating/patching process